x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.ts

```diff
@@ -8,8 +8,6 @@
 import _ from 'lodash';
 import { i18n } from '@kbn/i18n';
 // @ts-expect-error
-import { parse } from 'wellknown';
-// @ts-expect-error
 import turfCircle from '@turf/circle';
```

Comment:
nit: Can we remove the `@turf/circle` import here since we're not supporting `circle` types from Elasticsearch?